internal/config/config.go

```diff
@@ -11,6 +11,7 @@ import (
 type Config struct {
 	RunImages      []RunImage `toml:"run-images"`
 	DefaultBuilder string     `toml:"default-builder-image,omitempty"`
+	Experimental   bool       `toml:"experimental,omitempty"`
```

Comment:
We should test this, maybe along the lines of https://github.com/buildpacks/pack/blob/11e3a677ce38ee1166c96546171aa0b9877dbb37/internal/config/config_test.go#L47.